code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -213,3 +246,98 @@
 	else
 		owner.mind.faith.convertAct(owner, subject, B) // usr = preacher ; target = subject
 		return TRUE
+
+/datum/deconversion_ritual
+	var/datum/role/cultist/cultist = null
+	var/lol = FALSE
+	var/last_cultist = FALSE
+	var/success = DECONVERSION_ACCEPT
+
+/datum/deconversion_ritual/New(var/mob/living/deconverter, var/mob/living/deconvertee, var/obj/item/weapon/storage/bible/bible)
+	..()
+	if (!bible||!bible.my_rel||!deconverter||!deconvertee||!iscultist(deconvertee))
+		qdel(src)
+		return
+	var/mob/target
+	deconvertee.overlays += image('icons/effects/effects.dmi',src,"deconversion")
+	playsound(deconvertee, 'sound/effects/deconversion_start.ogg', 50, 0, -4)
+	cultist = deconvertee.mind.GetRole(CULTIST)
+	cultist.deconversion = src
+
+	deconvertee.eye_blurry = max(deconvertee.eye_blurry, 10)
+	deconvertee.Dizzy(30)
+	deconvertee.stuttering = max(deconvertee.stuttering, 10)
+	deconvertee.Jitter(30)
+	deconvertee.Knockdown(10)
+
+	if (istype(bible.my_rel, /datum/religion/cult))
+		lol = TRUE
```

Comment:
descriptive var names, I do get the joke as per below but it will help legibility